src/main/java/nextstep/ladder/ResultView.java

```diff
@@ -0,0 +1,36 @@
+package nextstep.ladder;
+
+import java.util.List;
+
+public class ResultView {
+
+    private static final String EXISTED_LINE = "|-----";
+    private static final String NOT_EXISTED_LINE = "|     ";
+
+    public void printPlayersName(Players players) {
+        System.out.println("실행 결과");
+        players.getPlayers()
+                .forEach(playerName -> {
+                    System.out.print(playerName + "    ");
+                });
+        System.out.println();
+    }
+
+    public void printLadders(Ladders ladders) {
+        ladders.getLines()
+                .stream()
+                .map(Line::getPoints)
+                .forEach(ResultView::printLines);
```

Comment:
```suggestion
                .forEach(this::printLines);
```

형태로 사용하시면 printLines를 static 하게 선언하지 않아도 됩니다.